templating/embedding_controllers.rst

```diff
@@ -79,7 +80,7 @@ string syntax for controllers (i.e. **bundle**:**controller**:**action**):
         {# ... #}
         <div id="sidebar">
             {{ render(controller(
-                'AppBundle:Article:recentArticles',
+                'App\\Controller\\ArticleController:recentArticles',
```

Comment:
Are the double-slashes needed? I'm really not sure - that's why I'm asking :)

Reply:
Twig auto-escapes the `\` character so by the time it makes it to the `controller` function PHP code, the namespace looks like `AppControllerArticleController`. By double escaping it, you're forcing the correct namespace to make it through `App\Controller\ArticleController`.